dev/com.ibm.ws.security.utility/resources/com/ibm/ws/security/utility/resources/UtilityOptions.nlsprops

```diff
@@ -286,4 +286,81 @@ tlsProfiler.option-desc.file=\
 tlsProfiler.option-key.verbose=\
 \ \ \ \ --verbose
 tlsProfiler.option-desc.verbose=\
-\tInclude additional information about unsafe protocols.
\ No newline at end of file
+\tInclude additional information about unsafe protocols.
+
+# -------- CONFIGURE FIPS OPTIONS --------------------------------------#
+# List of descriptive text for options. Use "\n\" for line continuations.
+# The code provides no whitespace: begin keys with "\ \ \ \ " for leading
+# whitespace, and begin descriptions with \t 
+#------------------------------\n at 72 chars -------------------------\n\#
+#------------------------------\n at 72 chars Leading "\ \ \ \ "-------\n\#
+#------------------------------\n at 72 chars Leading "\t"-------------\n\# 
+configureFIPS.desc=\
+\tConfigure FIPS 140-3 based on the Java installation configured by the\n\
+\tJAVA_HOME, JRE_HOME, or WLP_DEFAULT_JAVA_HOME environment variables\n\
+\tin that order. If none was configured, FIPS 140-3 will be configured\n\
+\tbased on the JVM used to the securityUtility command. The command\n\
```

Comment:
Is there a word missing here? e.g., used to _run_ the securityUtility command, perhaps?